File: evidence/corpus/curated/curated-store-lib/go/com/cervo/store/service/OrderService.go
package service

import "github.com/cervantesh/cervo-transpiler-java-to-go/evidence/corpus/curated/curated-store-lib/go/com/cervo/store/math"

func Total(subtotal int, tax int) int {
	return math.AddTax(subtotal, tax)
}
func DiscountedTotal(subtotal int, amount int) int {
	return math.Discount(subtotal, amount)
}

File: corpus/projects/curated-store-lib/src/main/java/com/cervo/store/service/OrderService.java
package com.cervo.store.service;

import com.cervo.store.math.PriceMath;

public class OrderService {
    public static int total(int subtotal, int tax) {
        return PriceMath.addTax(subtotal, tax);
    }

    public static int discountedTotal(int subtotal, int amount) {
        return PriceMath.discount(subtotal, amount);
    }
}
